src/cartesian/CartesianAxis.tsx

```diff
@@ -19,14 +19,21 @@ import {
 import { filterProps } from '../util/ReactUtils';
 import { getTicks } from './getTicks';
 
+/** The orientation of the axis in correspondence to the chart */
+export type Orientation = 'top' | 'bottom' | 'left' | 'right';
+/** A unit to be appended to a value */
```

Comment:
nit: a unit could also be a prefix (money)

Reply:
It theoretically could, but thats not how it is applied in recharts, right? We only append the unit as far as I can see.

Reply:
Yep, just maybe something to support in the future